lib/Sandbox.js

```diff
@@ -204,35 +207,49 @@ function getEnvironment() {
 function buildDependencyTree(
   baseDir: string,
   dependencySpecList: Array<string>,
-  packageDependencyTrace: Array<string>
+  context: {
```

Comment:
lol, I dislike such a common name in this specific case but still it's better than having a lot of positional args. Waiting to reimplement Esy in Reason.